scripts/create_farm.py

```diff
@@ -128,3 +238,14 @@ def make_row(y_ind, y_pos, num_trees_row):
 output_file.write(post_gen_info.read())
 post_gen_info.close()
 output_file.close()
+
+print("DONE")
+
+
+
+
+
+
+
+
+
```

Comment:
Remove extra new lines. Only one is needed at end of file.

Reply:
Done